prql-compiler/prqlc/src/cli.rs

```diff
@@ -27,6 +29,17 @@ pub fn main() -> color_eyre::eyre::Result<()> {
 
     if let Err(error) = cli.command.run() {
         eprintln!("{error}");
+        // TODO: this is clunky — we're using `backtrace.status()` to check if
+        // backtraces are enabled, but then printing the backtrace from the
+        // error. Is there a way to check whether backtrades are enabled without
+        // capturing another one; e.g. from the error itself?
```

Comment:
So this is equivalent to looking up the `RUST_BACKTRACE` env var?

Reply:
Ah, yes, probably. I thought this might be exposed directly, or encapsulated somewhere, but maybe we should just check that

Reply:
Looking at the source of `backtrace::Backtrace::enabled()`, it does check another env var, but we'd probably be find by just checking `RUST_BACKTRACE`.

Reply:
Ah, I somehow missed that — let's use that!